lighthouse-core/report/html/renderer/util.js

```diff
@@ -70,12 +70,39 @@ class Util {
     return output;
   }
 
+  /**
+   *
+   * @param {{score: number|null, scoreDisplayMode: string}} audit
+   */
+  static didAuditPass(audit) {
+    switch (audit.scoreDisplayMode) {
+      case 'manual':
+      case 'not-applicable':
+        return true;
+      case 'error':
+      case 'informative':
```

Comment:
so they show up as failing, if an informative audit has nothing to tell you it marks itself as `not-applicable` ;)

Reply:
aiight sg.

Reply:
should record this idea somewhere (to the lhr/lhr-lite description of these, maybe?)

Reply:
It's more an audit author concern since it's an immediate conclusion from the scoreDisplayMode that if an audit isn't `not-applicable` it means it was applicable :)

maybe in the custom audit docs?